gp-templates/locales.php

```diff
@@ -12,21 +12,32 @@
 		<?php _e( 'Filter:', 'glotpress' );?><input id="locales-filter" type="text" placeholder="<?php esc_attr_e('search', 'glotpress'); ?>" />
 	</div>
 
-	<table class="tablesorter locales">
+	<table class="tablesorter locales tablesorter-glotpress">
 		<thead>
-		<tr>
-			<th class="header"><?php _e( 'Name (in English)', 'glotpress' );?></th>
-			<th class="header"><?php _e( 'Native name', 'glotpress' );?></th>
-			<th class="header"><?php _e( 'Language code', 'glotpress' );?></th>
+		<tr class="tablesorter-headerRow">
+			<th class="header tablesorter-header tablesorter-headerAsc"><?php _e( 'Name (in English)', 'glotpress' );?></th>
+			<th class="header tablesorter-header tablesorter-headerUnSorted"><?php _e( 'Native name', 'glotpress' );?></th>
+			<th class="header tablesorter-header tablesorter-headerUnSorted"><?php _e( 'Language code', 'glotpress' );?></th>
 
 		</tr>
 		</thead>
 		<tbody>
-		<?php foreach ( $locales as $locale ) : ?>
-			<tr>
-				<?php echo "<td>" . gp_link_get( gp_url_join( gp_url_current(), $locale->slug ), $locale->english_name ) . "</td>" ?>
-				<?php echo "<td>" . gp_link_get( gp_url_join( gp_url_current(), $locale->slug ), $locale->native_name ) . "</td>" ?>
-				<?php echo "<td>" . gp_link_get( gp_url_join( gp_url_current(), $locale->slug ), $locale->slug ) . "</td>" ?>
+		<?php
+			$i = 0;
+
+			foreach ( $locales as $locale ) :
+				if ( $i % 2 ) {
+					$oe = 'even';
+				} else {
+					$oe = 'odd';
+				}
+
+				$i++;
+?>
```

Comment:
This can be simplified to something like

```php
$class = '';
foreach ( … ) {
    $class = ( 'odd' == $class ) ? 'even' : 'odd';
}
```

But: we don't need any PHP here. We can use the `:nth-child(odd)` selector in CSS instead.